src/macros.rs

```diff
@@ -113,7 +114,10 @@ macro_rules! redis_module {
             ctx: *mut $crate::raw::RedisModuleInfoCtx,
             for_crash_report: i32,
         ) {
-            $crate::base_info_func(ctx, for_crash_report == 1);
+            use $crate::InfoContext;
+            let mut __info_func__cb : Option<fn(&InfoContext, bool)> = None;
+            $( __info_func__cb = Some($info_func); )?
+            $crate::base_info_func(&$crate::InfoContext::new(ctx), for_crash_report == 1, __info_func__cb);
```

Comment:
@oshadmi no issues calling `base_info_func` when `__info_func__cb==None`?
Do we need to call `base_info_func` method if no `info_func` was passed?

Reply:
We need `base_info_func` to add panic/crash info - always.
The user info is optional, and when it is `None`, no info is added, except for panic/crash info.